npc/re/quests/quests_16_2_illusion.txt

```diff
@@ -4866,3 +4867,1824 @@ OnInit:
+			mes "And my treasure... ^FF0000Munch^000000, Munch... You brought^BDBDBDmy^000000treasure?";
+			next;
+			specialeffect EF_SCREEN_QUAKE;
+			mes "[An susceptible soldier]";
+			mes "^FF0000Give me^000000! My treasure!";
+			next;
+			mes "[An susceptible soldier]";
+			mes "Ha... Haha... Haha^BDBDBDha^000000ha...... Haha^BDBDBDha^000000hahaha......";
+			mes "It's shiny... shiny... It's pretty, so pretty.";
+			next;
+			mes "[An susceptible soldier]";
+			mes "Take this. I don't need it.";
+			mes "I have my treasure^BDBDBDnow^000000... It's shiny... Pretty.";
+			next;
+			mes " - ^424242As exchange, I took the " + getitemname(25271) + ". How could he have become obsessed with such empty garbage without seeing anything really valuable?^000000 -";
+			next;
+			mes " - ^424242Well, it might be just the idea of a living person that the " + getitemname(25271) + " is precious.^000000 -";
+			delitem 25316,5;	// Shabby_Metal_Chip
+			erasequest 16109;	// Vain Desire
+			setquest 16110;	// Vain Desire - Cooldown
+			getitem 25271,1;	// IllusionStone
+			getexp 1200000,1000000;	// todo unknown exp (irowiki values)
+			close;
+		}
+		mes "[An susceptible soldier]";
+		mes "Munch, Murmur... ^FF0000munch^000000, munch.^FF0000munch^000000...";
+		mes "The Damn^BDBDBDhuh^000000Mon..^BDBDBDDmonster^000000...";
+		next;
+		mes "[An susceptible soldier]";
+		mes "It's all mine ^BDBDBDCould be^000000 all^BDBDBDmine^000000...";
+		mes "My... my treasure^BDBDBDall^000000of...my... ^FF0000munch,munch^000000...";
+		next;
+		select( "What treasure?", "What monster?" );	// note: the selection doesn't change the dialog
+		mes "[An susceptible soldier]";
+		mes "The Damn^BDBDBDhuh^000000Mon.. ^FF0000" + strmobinfo(1,3799) + " monster^000000...";
+		mes "They take^BDBDBDmy^000000treasure^BDBDBDD.... My!! treasure^000000...!!";
+		next;
+		mes "[An susceptible soldier]";
+		mes "Munch, Murmur... ^FF0000munch^000000, munch.";
+		mes "Munch,  ^FF0000Munch^000000... Murmur...";
+		next;
+		if (select( "Let me find.", "Okay." ) == 2) {
+			mes "[An susceptible soldier]";
+			mes "Yes... It's mine^BDBDBDhuh^000000... My^BDBDBDtreasure^000000...";
+			mes "I couldn't have ^BDBDBDit^000000... The Damn ^BDBDBDmon^000000monsters...";
+			mes "Munch, ^FF0000Munch Munch Munch^000000...";
+			close;
+		}
+		mes "[An susceptible soldier]";
+		mes "You? Are^BDBDBDyou^000000 going?";
+		mes "Go ^BDBDBDgo^000000get it! My^BDBDBDtreasure^000000! You can find it from them! My^BDBDBDtreasure^000000!";
+		next;
+		mes "[An susceptible soldier]";
+		mes "And payback to " + strmobinfo(1,3799) + ". ^FF0000Revenge^000000... Munch Munch... My^BDBDBDrevenge^000000.. Please...";
+		mes "Get rid of 10... monsters! Munch, ^FF0000Munch Munch^000000... Get ^BDBDBDrid^000000of them!!!";
+		next;
+		mes " - ^424242Narin said, I can get some ^0000CD" + getitemname(25316) + "^424242 from here, it would be enough. ^0000CDLet me prepare^424242 5 pieces.^000000 - ";
```

Comment:
```suggestion
		mes "[Susceptible Soldier]";
		mes "And payback to " + strmobinfo(1,3799) + ". ^FF0000Revenge^000000... Munch munch... My ^BDBDBDrevenge^000000.. Please...";
		mes "Get rid of 10... monsters! Munch, ^FF0000munch munch^000000... Get ^BDBDBDrid^000000 of them!!!";
		next;
		mes " - ^424242Narin said, I can get some ^0000CD" + getitemname(25316) + "^424242 from here. It would be enough for him. ^0000CDLet me prepare^424242 5 pieces.^000000 - ";
```